views/lib/forms/form-auto-logout-alert.hbs

```diff
@@ -0,0 +1,47 @@
+<style>
+    .sloth {
+        width: 35%; 
+        margin-bottom: -50px;
+        height: auto; 
+        opacity: 0.6;
+    }
+
+    .sloth-text {
+        text-align: center;
+        padding: 15px;
+    }
+
+    @media (min-width: 575.98px){
+    .sloth {
+        float: right;
+        margin-top: -12px;
+        margin-bottom: -20px;
+        text-align: center;
+    }
+
+    .sloth-text {
+        text-align: left;
+        padding: 35px 0 10px 28px;
+    }
+    }
+</style>
+<div class="form-group" data-rst-default="{{JWT_TIMEOUT_SECONDS}}">
+<div class="text-center">
+    <img src="/images/sloth.svg" class="sloth"/>
+    <div class="sloth-text">
+        Achtung, du wirdst in <br>
+        <span style="color: red; font-size: larger;"><span class="js-time">n/a</span> Minuten</span> automatisch Ausgeloggt! <br>
+        Klicke OK und deine Sitzung wird um weitere zwei Stunden verlänegrt.
+    </div>
+    <div class="sloth-text sloth-text-error">
+        </br>
+        Upps... das hätte nich passieren dürfen! </br> 
+        Deine Sitzung konnte nicht verlängert werden. Versuchen wir es gleich nochmal
+    </div>
+</div>
+<div style="text-align: center; padding: 0 8px; margin-bottom: 30px;">
+    <button class="btn btn-primary btn-add btn-incRst" style="width: 95%;">
```

Comment:
```suggestion
    <button type="button" class="btn btn-primary btn-add btn-incRst" style="width: 95%;"><
```
Buttons have the default type "submit" which will reload the page. You solved it with js, but correcting the button is still a good idea I think.

Reply:
👍 really good suggestion